APMrover2/wscript

```diff
@@ -22,6 +22,10 @@ def build(bld):
             'AP_VisualOdom',
             'AP_AdvancedFailsafe',
             'AP_WheelEncoder',
+            'AC_Fence',
+            'AP_InertialNav',
+            'AC_Fence',
```

Comment:
arf, I haven't rebase on the corrected previous PR. It will be clearer when other will be merged. I will correct that tomorrow